### PR TITLE
docs: adds tabs to elements docs

### PR DESCRIPTION
**Pull request by langz**
This PR adds the following tabs to all [elements](https://eufemia.dnb.no/uilib/elements/)(except for Code).

Please take a quick look through the following docs for elements:
https://eufemia-git-chore-add-tabs-to-elements-docs-eufemia.vercel.app/uilib/elements/

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/elements/horizontal-rule.mdx [hunk 1]
```diff
@@ -1,40 +1,16 @@
 ---
 title: 'Hr-line (Divider)'
+description: 'The `<hr />` tag in HTML stands for horizontal rule and is used to insert a horizontal rule or a thematic break in an HTML page to divide or separate document sections.'
+showTabs: true
+theme: 'sbanken'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/elements/horizontal-rule.mdx:5]:
```suggestion
```

File: packages/dnb-design-system-portal/src/docs/uilib/elements/image.mdx [hunk 2]
```diff
@@ -1,38 +1,13 @@
 ---
 title: 'Image'
+showTabs: true
+theme: 'sbanken'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/elements/image.mdx:4]:
```suggestion
```

File: packages/dnb-design-system-portal/src/docs/uilib/elements/lists.mdx [hunk 3]
```diff
@@ -1,92 +1,13 @@
 ---
 title: 'Lists'
+showTabs: true
+theme: 'sbanken'
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/elements/lists.mdx:4]:
```suggestion
```

**Review by joakbjerk — APPROVED** [review]
Looks good to me 🙇

**Pull request merged**